integration-tests/azure-keyvault/src/main/java/io/quarkiverse/azure/keyvault/secret/it/KeyVaultSecretConfigResource.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkiverse.azure.keyvault.secret.it;
+
+import static jakarta.ws.rs.core.MediaType.TEXT_PLAIN;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Path("/keyvaultConfig")
+public class KeyVaultSecretConfigResource {
+    @ConfigProperty(name = "kv//secret1")
+    String value = "mysecret";
```

Comment:
The default value `mysecret` is same as the secret value for `secret1`, suggest change to another different value to differentiate two test cases:
* Specified secret (`secret1`) exists in key vault: return value of secret `secret1`
* Specified secret doesn't exist in key vault: return default value defined for variable `value`

Reply:
Remove the default value. We cannot start the app if the required property does exist.